### PR TITLE
Fix iOS Build Errors

### PR DESCRIPTION
**Pull request by nibanks**
## Description

Fixes a recent macOS build error. Not sure why it's just now showing up.

## Testing

Automated CI

## Documentation

N/A

### CODE REVIEW

**Comment by nibanks** [comment]
This will need to be backported to release branches that build iOS.

**Review by csujedihy — APPROVED** [review]

File: src/test/bin/quic_gtest.cpp [hunk 1]
```diff
@@ -1116,6 +1116,7 @@ TEST_P(WithHandshakeArgs4, RandomLoss) {
             (uint8_t)GetParam().ServerStatelessRetry,
             0,  // ClientUsesOldVersion
             0,  // MultipleALPNs
+            0,  // GreaseQuicBitExtension
```
Comment (anrossi) [inline src/test/bin/quic_gtest.cpp:1119]: How did this even build with a parameter number mismatch?

File: src/inc/msquic_posix.h [hunk 2]
```diff
@@ -505,7 +505,7 @@ QuicAddrToString(
             Address[0] = ']';
             Address++;
         }
-        sprintf(Address, ":%hu", ntohs(Addr->Ipv4.sin_port));
+        snprintf(Address, 64, ":%hu", ntohs(Addr->Ipv4.sin_port));
```
Comment (anrossi) [inline src/inc/msquic_posix.h:508]: No constant available for this?

File: src/test/bin/quic_gtest.cpp (hunk 1 above)
Comment (nibanks) [reply src/test/bin/quic_gtest.cpp:1119]: It just didn't initialize the last value.

File: src/inc/msquic_posix.h (hunk 2 above)
Comment (nibanks) [reply src/inc/msquic_posix.h:508]: It's just defined right above. I don't think it's needed.

**Pull request merged**